### PR TITLE
Fixed 2 coverflow issues

### PR DESCRIPTION
**Pull request by DohyungLim**
1. https://github.com/Samsung/TAU-Design-Editor/issues/435
2. https://github.com/Samsung/TAU-Design-Editor/issues/438

- Add the missing tau.circle.css and style.css file line.

### CODE REVIEW

File: examples/wearable/UIComponents/contents/interactive/coverflow/index.html [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <head>
 	<meta name="viewport" content="width=device-width,user-scalable=no"/>
 	<link rel="stylesheet" href="../../../lib/tau/wearable/theme/default/tau.css">
+	<link rel="stylesheet" href="../../../lib/tau/wearable/theme/default/tau.circle.css">
```
Comment (singa2000) [inline examples/wearable/UIComponents/contents/interactive/coverflow/index.html:6]: I think it's better to add
	<link href="../../lib/tau/wearable/theme/default/tau.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
	<link href="../../css/style.css" rel="stylesheet" />
	<link href="../../css/style.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />

File: examples/mobile/UIComponents/components/interactive/coverflow/index.html [hunk 2]
```diff
@@ -14,13 +14,13 @@ <h1>CoverFlow</h1>
...
 		<div class="ui-coverflow" data-effect="carousel">
 			<ul class="flip-items">
-				<li><img src="images/1.jpg" class="coverflow-image"></li>
-				<li><img src="images/2.jpg" class="coverflow-image"></li>
-				<li><img src="images/3.jpg" class="coverflow-image"></li>
-				<li><img src="images/4.jpg" class="coverflow-image"></li>
-				<li><img src="images/5.jpg" class="coverflow-image"></li>
-				<li><img src="images/6.jpg" class="coverflow-image"></li>
-				<li><img src="images/7.jpg" class="coverflow-image"></li>
+				<li><img src="images/1.jpg" width="150px" height="150px"></li>
```
Comment (lslachciak) [inline examples/mobile/UIComponents/components/interactive/coverflow/index.html]: This is defined in style.css, right?
So how about referencing to it like other apps do
e.g.
https://github.com/Samsung/TAU/blob/tau_1.1/examples/wearable/UIComponents/contents/graphs/static_graph.html
`	<link href="../../css/style.css" rel="stylesheet" />`

**Review by lslachciak — COMMENTED** [review]
small comment in code,
Verified, mobile coverflow works well in live preview
in case of wearable when using mouse scroller there is one-time small bump up

**Comment by lslachciak** [comment]
Links for testing PR locally:
(need to have running WATT instance with PR checked out in TAU/1.1 WATT's directory)
http://localhost:3000/demos?path=1.1%2Fexamples%2Fmobile%2FUIComponents%2Fcomponents%2Finteractive%2Fcoverflow%2Findex.html

http://localhost:3000/demos?path=1.1%2Fexamples%2Fwearable%2FUIComponents%2Fcontents%2Finteractive%2Fcoverflow%2Findex.html

File: examples/mobile/UIComponents/components/interactive/coverflow/index.html (hunk 2 above)
Comment (DohyungLim) [reply examples/mobile/UIComponents/components/interactive/coverflow/index.html]: sure. no problem.

**Review by lslachciak — APPROVED** [review]
Reported issues are fixed.
On wearable there is small unexpected scroll up when using mouse scroller, but this can be part of another issue

**Comment by DohyungLim** [comment]
ok. thanks for your review.
I will merge it.

**Comment by lslachciak** [comment]
The last coverflow issue is reported here: https://github.com/Samsung/TAU-Design-Editor/issues/442

**Pull request merged**